src/main/java/de/numcodex/feasibility_gui_backend/dse/DseService.java

```diff
@@ -0,0 +1,65 @@
+package de.numcodex.feasibility_gui_backend.dse;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.numcodex.feasibility_gui_backend.dse.api.DseProfile;
+import de.numcodex.feasibility_gui_backend.dse.api.DseProfileTreeNode;
+import de.numcodex.feasibility_gui_backend.dse.persistence.DseProfileRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@Slf4j
+public class DseService {
+
+  @Value("classpath:de/numcodex/feasibility_gui_backend/dse/profile_tree.json")
```

Comment:
the profile_tree.json should be part of the ontology and be supplied as such

Reply:
Is there an issue in the ontology generator that could be linked here?